package.json

```
@@ -10,35 +10,46 @@
     "react-dom": "^19.0.0",
     "react-dropzone": "^14.3.8",
     "react-icons": "^5.5.0",
     "react-markdown": "^10.1.0",
     "react-router": "^7.5.0",
+    "react-toastify": "^11.0.5",
     "remark-breaks": "^4.0.0",
+    "remark-docx": "^0.1.6",
     "remark-gfm": "^4.0.1",
     "tailwind-merge": "^3.2.0",
     "tw-animate-css": "^1.2.5",
+    "zod": "^3.24.3",
```

Biome (1.9.4)

[error] The key zod was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)